src/ResourceTypeRegistry.js

```diff
@@ -5,7 +8,23 @@
  * whose property is being retrieved/set, and the value to set it to, if any.
  */
 const autoGetterSetterProps = ["dbAdapter", "beforeSave", "beforeRender",
-  "labelMappers", "defaultIncludes", "info", "parentType"];
+  "labelMappers", "defaultIncludes", "info", "parentType", "behaviors"];
```

Comment:
I don't think we can add an automatic getter/setter for `behaviors`, because every time `behaviors` is set, the `_inverseExceptions` may need to be updated, which they won't be with the automated logic. So, instead, `behaviors` can become a custom getter/setter method like `urlTemplates` is, and the `_inverseExceptions` logic can move there instead of being in `type`.